### DISCUSSION TITLE
绘制组件的编辑按钮是由什么控制显示隐藏的呢，能不能设置为我绘制一个新的点 编辑组件就显示出来

### DISCUSSION THREAD

**Discussion by jiege15121313** [zouyaoji/vue-cesium#302] [General]
绘制组件的编辑按钮是由什么控制显示隐藏的呢，能不能设置为我绘制一个新的点 编辑组件就显示出来

**Comment by zouyaoji** [comment]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/analyses/vc-drawings 文档中写了的


![image](https://user-images.githubusercontent.com/24430116/199004541-900d71af-c572-4832-96c0-bc17f9347b6c.png)
